contracts/voting-verifier/src/execute.rs

```diff
@@ -58,7 +58,7 @@ pub fn verify_worker_set(
                 source_chain: config.source_chain,
                 source_gateway_address: config.source_gateway_address,
                 confirmation_height: config.confirmation_height,
-                expires_at: env.block.height + config.block_expiry,
+                expires_at: env.block.height.saturating_add(config.block_expiry),
```

Comment:
again, I'd rather use the checked version here because otherwise the expiry interval could have an unexpected length

Reply:
I kind of feel the same way as here: https://github.com/axelarnetwork/axelar-amplifier/pull/296#discussion_r1525694763

Reply:
see my response there